docs/docs/contributing/getting-started.md

```diff
@@ -83,9 +83,22 @@ Generating Documentation
 -------------------------
 To generate this page and other static page docs, run
 ```bash
+# the generated docs will be located in scaladoc/output
 $ sbt
-> genDocs
+> scalaDoc/generateScalaDocumentation
+
+# or run the following bash script to generate docs under $PWD/docs/_site
+$ ./project/scripts/genDocs
```

Comment:
I'd avoid mentioning this script, it seems a bit more internal than the `generateScalaDocumentation` task.